### PR TITLE
fix(dedicated-cloud.upgrade): switch datastore from hourly to monthly

### PR DESCRIPTION
**Pull request by radireddy**
use exact plan code while getting plan details from datastore service options

ref #DTRSD-23106

| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | DTRSD-23106
| License          | BSD 3-Clause

## Description

Use exact plan code while getting plan details from datastore service options

### CODE REVIEW

**Review by antleblanc — CHANGES_REQUESTED** [review]
Could you please use signing commit.

**Review by Heiiji — APPROVED** [review]

**Review by marie-j — APPROVED** [review]

**Review by JeremyDec — DISMISSED** [review]

**Review by cbourgois — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/resource/upgrade/upgrade.controller.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ export default class {
   }
 
   fetchCatalog() {
+    console.log('fetchCatalog');
```
Comment (mohammed-zahaf) [inline packages/manager/apps/dedicated/client/app/components/dedicated-cloud/resource/upgrade/upgrade.controller.js:34]: Probably missed

**Comment by ovh-cds** [comment]
CDS Report translator-push#13869.0 ✘  
* 
  * push ✘

**Comment by antleblanc** [comment]
Rewriting history (first commit)

**Comment by antleblanc** [comment]
History has been rewrite successfully.

**Review by antleblanc — APPROVED** [review]

**Pull request merged**